packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -181,6 +184,12 @@ class TabBarTheme extends InheritedTheme with Diagnosticable {
   /// please use the [TabBarThemeData.indicatorAnimation] property in [data] instead.
   TabIndicatorAnimation? get indicatorAnimation => _data != null ? _data.indicatorAnimation : _indicatorAnimation;
 
+  /// Overrides the default value for [TabBar.splashBorderRadius].
+  ///
+  /// This property is obsolete and will be deprecated in a future release:
+  /// please use the [TabBarThemeData.splashBorderRadius] property in [data] instead.
+  BorderRadius? get splashBorderRadius => _data != null ? _data.splashBorderRadius : _splashBorderRadius;
```

Comment:
Do you think we could add this property to the `TabBarThemeData` without introducing an obsolete field in `TabBarTheme`?